firebaseMsgSwGenerator.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
I renamed the generator. I think we won't need more generators anytime soon, so just keeping the specific one instead of making it more generic. If another one is added, we should probably make it generic.